### PR TITLE
Updated according to changes in eventuate common

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: eventuate-messaging-partition-management-tests/build.gradle [hunk 1]
```diff
@@ -1,6 +1,8 @@
 apply plugin: PublicModulePlugin
 
 dependencies {
+    compile 'com.google.collections:google-collections:1.0'
```
Comment (cer) [inline eventuate-messaging-partition-management-tests/build.gradle]: Why is this dependency needed? There are not other changes in this module?

Comment (dartartem) [reply eventuate-messaging-partition-management-tests/build.gradle]: eventuate-common-coordination-leadership does not contain 'org.apache.curator:curator-recipes' dependency anymore. It is moved to  eventuate-common-coordination-leadership-zookeeper. But there were used immutable collections which curator-recipes provided indirectly.

Comment (cer) [reply eventuate-messaging-partition-management-tests/build.gradle]: Not sure that this dependency is really needed. e.g. `Arrays.asList`

Comment (dartartem) [reply eventuate-messaging-partition-management-tests/build.gradle]: ok, I will fix

**Comment by cer** [comment]
@dartartem this PR cannot be merged. There are conflicts

**Comment by dartartem** [comment]
> this PR cannot be merged. There are conflicts

Strange, I will fix

**Pull request merged**